src/Common/tests/System/AssertExtensions.cs

```diff
@@ -13,5 +14,17 @@ public static void Throws<T>(Action action, string message)
         {
             Assert.Equal(Assert.Throws<T>(action).Message, message);
         }
+
+        [Conditional("DEBUG")]
+        public static void ThrowsInDebug<T>(Action action) where T : Exception => Assert.Throws<T>(action);
+
+        [Conditional("DEBUG")]
+        public static void ThrowsInDebug<T>(Func<object> func) where T : Exception => Assert.Throws<T>(func);
+
+        [Conditional("DEBUG")]
+        public static void ThrowsAnyInDebug<T>(Action action) where T : Exception => Assert.ThrowsAny<T>(action);
+
+        [Conditional("DEBUG")]
+        public static void ThrowsAnyInDebug<T>(Func<object> func) where T : Exception => Assert.ThrowsAny<T>(func);
```

Comment:
That's true for your usage, but these members were added to an existing file in Common that's meant for other test suites to include.  They're also odd in nature, relying on the fact that the current Asserts throw exceptions as a way of indicating a failed assertion. I don't think there's enough value to add these.  The ThrowsInDebug members are also never used.  I would just remove them and the call sites.